src/sb_pointcloud_processing/scripts/kill_pcl.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
Quick doc comment explaining what this script does, why it's here, and why it's necessary would be helpful.